templates/repo/issue/view_content/pull.tmpl

```diff
@@ -131,24 +131,25 @@
 		{{$showGeneralMergeForm := false}}
 		<div class="ui attached merge-section segment {{if not $.LatestCommitStatus}}no-header{{end}}">
 			{{if .Issue.PullRequest.HasMerged}}
-				<div class="item text">
-					{{if .Issue.PullRequest.MergedCommitID}}
-						{{$link := printf "%s/commit/%s" $.Repository.Link (.Issue.PullRequest.MergedCommitID|PathEscape)}}
-						{{if eq $.Issue.PullRequest.Status 3}}
-							{{$.locale.Tr "repo.pulls.manually_merged_as" ($link|Escape) (ShortSha .Issue.PullRequest.MergedCommitID) | Safe}}
+				<div class="item item-section text gt-f1">
+					<div class="item-section-left">
+						{{if .Issue.PullRequest.MergedCommitID}}
+							{{$link := printf "%s/commit/%s" $.Repository.Link (.Issue.PullRequest.MergedCommitID|PathEscape)}}
+							{{if eq $.Issue.PullRequest.Status 3}}
+								{{$.locale.Tr "repo.pulls.manually_merged_as" ($link|Escape) (ShortSha .Issue.PullRequest.MergedCommitID) | Safe}}
+							{{else}}
+								{{$.locale.Tr "repo.pulls.merged_as" ($link|Escape) (ShortSha .Issue.PullRequest.MergedCommitID) | Safe}}
+							{{end}}
 						{{else}}
-							{{$.locale.Tr "repo.pulls.merged_as" ($link|Escape) (ShortSha .Issue.PullRequest.MergedCommitID) | Safe}}
+							{{$.locale.Tr "repo.pulls.has_merged"}}
 						{{end}}
-					{{else}}
-						{{$.locale.Tr "repo.pulls.has_merged"}}
+					</div>
+					{{if .IsPullBranchDeletable}}
+						<div class="item-section-right">
+							<button class="delete-button ui button" data-url="{{.DeleteBranchLink}}">{{$.locale.Tr "repo.branch.delete_html"}}</a>
```

Comment:
`<button>…</a>`?

Reply:
By the way, isn't it possible to inline this `div` into the button simply by giving the button the `item-section-right` class?
Or is the CSS selector too restrictive?

Reply:
Ah, good catch.